### ISSUE TITLE
In BCK, Migrations from 0.15 to 0.16 Fail

### ISSUE THREAD

**Issue by DXCanas** [learningequality/kolibri#10101]
## Observed behavior

Failed migration

## Errors and logs

```
$ kolibri manage migrate
INFO: Initialized plugins.json
INFO     2023-02-08 22:50:53,194 Option CACHE_BACKEND in section [Cache] being overridden by environment variable KOLIBRI_CACHE_BACKEND
INFO     2023-02-08 22:50:53,194 Option CACHE_PASSWORD in section [Cache] being overridden by environment variable KOLIBRI_CACHE_PASSWORD
INFO     2023-02-08 22:50:53,194 Option CACHE_LOCATION in section [Cache] being overridden by environment variable KOLIBRI_CACHE_LOCATION
INFO     2023-02-08 22:50:53,195 Option DATABASE_ENGINE in section [Database] being overridden by environment variable KOLIBRI_DATABASE_ENGINE
INFO     2023-02-08 22:50:53,195 Option DATABASE_NAME in section [Database] being overridden by environment variable KOLIBRI_DATABASE_NAME
INFO     2023-02-08 22:50:53,195 Option DATABASE_PASSWORD in section [Database] being overridden by environment variable KOLIBRI_DATABASE_PASSWORD
INFO     2023-02-08 22:50:53,195 Option DATABASE_USER in section [Database] being overridden by environment variable KOLIBRI_DATABASE_USER
INFO     2023-02-08 22:50:53,195 Option DATABASE_HOST in section [Database] being overridden by environment variable KOLIBRI_DATABASE_HOST
INFO     2023-02-08 22:50:53,195 Option DATABASE_PORT in section [Database] being overridden by environment variable KOLIBRI_DATABASE_PORT
INFO     2023-02-08 22:50:53,195 Option CHERRYPY_START in section [Server] being overridden by environment variable KOLIBRI_CHERRYPY_START
WARNING  2023-02-08 22:50:53,195 Option CHERRYPY_START in section [Server] is deprecated, please remove it from your options.ini file
INFO     2023-02-08 22:50:53,196 Option DEBUG in section [Server] being overridden by environment variable KOLIBRI_DEBUG
INFO     2023-02-08 22:50:53,196 Option DEBUG_LOG_DATABASE in section [Server] being overridden by environment variable KOLIBRI_DEBUG_LOG_DATABASE
INFO     2023-02-08 22:50:53,196 Option RUN_MODE in section [Deployment] being overridden by environment variable KOLIBRI_RUN_MODE
INFO     2023-02-08 22:50:53,196 Option LANGUAGES in section [Deployment] being overridden by environment variable KOLIBRI_LANGUAGES
INFO     2023-02-08 22:50:53,196 Option ZIP_CONTENT_ORIGIN in section [Deployment] being overridden by environment variable KOLIBRI_ZIP_CONTENT_ORIGIN
INFO     2023-02-08 22:50:53,196 Option REMOTE_CONTENT in section [Deployment] being overridden by environment variable KOLIBRI_REMOTE_CONTENT
WARNING  2023-02-08 22:50:53,204 Option CHERRYPY_START in section [Server] is deprecated, please remove it from your options.ini file
INFO     2023-02-08 22:50:53,207 Option CACHE_BACKEND in section [Cache] being overridden by environment variable KOLIBRI_CACHE_BACKEND
INFO     2023-02-08 22:50:53,207 Option CACHE_PASSWORD in section [Cache] being overridden by environment variable KOLIBRI_CACHE_PASSWORD
INFO     2023-02-08 22:50:53,208 Option CACHE_LOCATION in section [Cache] being overridden by environment variable KOLIBRI_CACHE_LOCATION
INFO     2023-02-08 22:50:53,208 Option DATABASE_ENGINE in section [Database] being overridden by environment variable KOLIBRI_DATABASE_ENGINE
INFO     2023-02-08 22:50:53,208 Option DATABASE_NAME in section [Database] being overridden by environment variable KOLIBRI_DATABASE_NAME
INFO     2023-02-08 22:50:53,208 Option DATABASE_PASSWORD in section [Database] being overridden by environment variable KOLIBRI_DATABASE_PASSWORD
INFO     2023-02-08 22:50:53,208 Option DATABASE_USER in section [Database] being overridden by environment variable KOLIBRI_DATABASE_USER
INFO     2023-02-08 22:50:53,208 Option DATABASE_HOST in section [Database] being overridden by environment variable KOLIBRI_DATABASE_HOST
INFO     2023-02-08 22:50:53,208 Option DATABASE_PORT in section [Database] being overridden by environment variable KOLIBRI_DATABASE_PORT
INFO     2023-02-08 22:50:53,208 Option CHERRYPY_START in section [Server] being overridden by environment variable KOLIBRI_CHERRYPY_START
WARNING  2023-02-08 22:50:53,209 Option CHERRYPY_START in section [Server] is deprecated, please remove it from your options.ini file
INFO     2023-02-08 22:50:53,209 Option DEBUG in section [Server] being overridden by environment variable KOLIBRI_DEBUG
INFO     2023-02-08 22:50:53,209 Option DEBUG_LOG_DATABASE in section [Server] being overridden by environment variable KOLIBRI_DEBUG_LOG_DATABASE
INFO     2023-02-08 22:50:53,209 Option RUN_MODE in section [Deployment] being overridden by environment variable KOLIBRI_RUN_MODE
INFO     2023-02-08 22:50:53,209 Option LANGUAGES in section [Deployment] being overridden by environment variable KOLIBRI_LANGUAGES
INFO     2023-02-08 22:50:53,209 Option ZIP_CONTENT_ORIGIN in section [Deployment] being overridden by environment variable KOLIBRI_ZIP_CONTENT_ORIGIN
INFO     2023-02-08 22:50:53,209 Option REMOTE_CONTENT in section [Deployment] being overridden by environment variable KOLIBRI_REMOTE_CONTENT
INFO     2023-02-08 22:50:53,574 Running Kolibri with the following settings: kolibri.deployment.default.settings.cloud_kolibri_settings
INFO     2023-02-08 22:50:53,652 Version was 0.15.9, new version: 0.16.0.dev0+git.20230208155958
INFO     2023-02-08 22:50:53,652 Running update routines for new version...
Operations to perform:
  Apply all migrations: admin, analytics, auth, bookmarks, content, contenttypes, device, discovery, exams, kolibriauth, lessons, logger, morango, notifications, sessions
Running migrations:
  No migrations to apply.
Operations to perform:
  Apply all migrations: admin, analytics, auth, bookmarks, content, contenttypes, device, discovery, exams, kolibriauth, lessons, logger, morango, notifications, sessions
Running migrations:
  No migrations to apply.
Installed 2 object(s) from 1 fixture(s)
Error: Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 1900, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.DuplicateTable: relation "queue__scheduled_time" already exists
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/kolibri/utils/cli.py", line 193, in invoke
    initialize(**get_initialize_params())
  File "/usr/local/lib/python3.9/site-packages/kolibri/utils/main.py", line 299, in initialize
    update(version, kolibri.__version__)
  File "/usr/local/lib/python3.9/site-packages/kolibri/utils/main.py", line 365, in update
    run_upgrades(old_version, new_version)
  File "/usr/local/lib/python3.9/site-packages/kolibri/core/upgrade.py", line 154, in run_upgrades
    filter(filter_upgrade, get_upgrades(app_configs=app_configs))
  File "/usr/local/lib/python3.9/site-packages/kolibri/core/upgrade.py", line 126, in get_upgrades
    version_upgrades += [
  File "/usr/local/lib/python3.9/site-packages/kolibri/core/upgrade.py", line 130, in <listcomp>
    if isinstance(upgrade, VersionUpgrade)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/django/utils/functional.py", line 238, in inner
    self._setup()
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/django/utils/functional.py", line 386, in _setup
    self._wrapped = self._setupfunc()
  File "/usr/local/lib/python3.9/site-packages/kolibri/core/tasks/main.py", line 18, in __job_storage
    return Storage(
  File "/usr/local/lib/python3.9/site-packages/kolibri/core/tasks/storage.py", line 87, in __init__
    self.Base.metadata.create_all(self.engine)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/schema.py", line 4930, in create_all
    bind._run_ddl_visitor(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 3228, in _run_ddl_visitor
    conn._run_ddl_visitor(visitorcallable, element, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 2211, in _run_ddl_visitor
    visitorcallable(self.dialect, self, **kwargs).traverse_single(element)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/visitors.py", line 524, in traverse_single
    return meth(obj, **kw)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/ddl.py", line 851, in visit_metadata
    self.traverse_single(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/visitors.py", line 524, in traverse_single
    return meth(obj, **kw)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/ddl.py", line 907, in visit_table
    self.traverse_single(index, create_ok=True)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/visitors.py", line 524, in traverse_single
    return meth(obj, **kw)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/ddl.py", line 938, in visit_index
    self.connection.execute(CreateIndex(index))
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 1380, in execute
    return meth(self, multiparams, params, _EMPTY_EXECUTION_OPTS)
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/sql/ddl.py", line 80, in _execute_on_connection
    return connection._execute_ddl(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 1472, in _execute_ddl
    ret = self._execute_context(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 1943, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 2124, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/util/compat.py", line 210, in raise_
    raise exception
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/base.py", line 1900, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.9/site-packages/kolibri/dist/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.DuplicateTable) relation "queue__scheduled_time" already exists
[SQL: CREATE INDEX queue__scheduled_time ON jobs (queue, scheduled_time)]
(Background on this error at: https://sqlalche.me/e/14/f405)
```

## User-facing consequences

Deploying upgrades from 0.15 to 0.16 fail

## Steps to reproduce

Set up an 0.15.9 kolibri instance (preferably in BCK) with a Postgres/psycopg db, then try and install an 0.16 WHL on top of it

## Context

BCK, currently runs Kolibri in the Python 3.9 Ubuntu Buster docker image with Postgres 13

**Comment by DXCanas** [comment]
cc @crc8109

**Comment by rtibbles** [comment]
This was fixed in #10667 by @jredrejo.
